tutorials/outflux.md

```diff
@@ -0,0 +1,174 @@
+                  with the discovered schema
+```
+
+You will see the help output for Outflux, a brief explanation of what it can do, the usage, and available commands. 
+
+For instructions on how to set up Outflux from source you can visit the [README](https://github.com/timescale/outflux/blob/master/README.md).
+
+## Importing Sample Data
+
+If you don’t already have an existing InfluxDB database, you can try Outflux by importing this example file with data written in the Influx Line Protocol found at https://timescaledata.blob.core.windows.net/datasets/outflux_taxi.txt 
+
+You can use the Influx CLI client to load the data. The file will first create the “outflux_tutorial” database and then do 2741 inserts. 
+
+```bash
+$ influx -import -path=outflux_taxit.txt -database=outflux_tutorial
+2019/03/27 11:39:11 Processed 1 commands
+2019/03/27 11:39:11 Processed 2741 inserts
+2019/03/27 11:39:11 Failed 0 inserts
+```
+
+The data in the file is without a timestamp so the current time of the Influx server will be used at the time of insert. All the data points belong to one measurement `taxi`. The points are tagged with location, rating and vendor. Four fields are recorded: fare, mta_tax, tip and tolls. 
+The `influx` client assumes the server is available at http://localhost:8086 by default.
+
+## Schema Discovery, Validation and Transfer
+
+One of Outflux’s features is the ability to discover the schema of an InfluxDB measurement and either validate that a TimescaleDB table exists that can hold the transferred data, or create a new table that can satisfy the schema requirements. 
+
+We can now create a TimescaleDB hypertable ready to receive the demo data we inserted into the InfluxDB instance. If you followed the tutorial and inserted the data from the example, there should be a `taxi` measurement in the `outflux_tutorial` database in the InfluxDB instance. 
+
+The `schema-transfer` command of Outflux can work with 4 schema strategies: 
+* `ValidateOnly`: checks if a specified TimescaleDB database has a hypertable with the proper columns and if it’s partitioned properly, but will not perform modifications
+* `CreateIfMissing`: validates everything if it exists, and it will check if the TimescaleDB  extension is installed (if the hypertable is missing, it will create it  and partition it properly)
+* `DropAndCreate`: drops any existing table with the same name as the measurement, and creates a new hypertable and partitions it properly
+* `DropCascadeAndCreate`: performs the same action as DropAndCreate with the additional strength of executing a cascade table drop if there is an existing table with the same name as the measurement
+
+The `schema-transfer` can be called with parameters that specify the schema strategy, and if the user wants the tags or fields of a measure to be transferred as a single JSONB column. By default each tag and each field is created as a separate column. 
```

Comment:
Either "The `schema-transfer` _command_ can be..." or "`Schema-transfer` can be..."